python/arcticdb/adapters/lmdb_library_adapter.py

```diff
@@ -168,24 +168,7 @@ def get_library_config(self, name, library_options: LibraryOptions):
         )
 
     def cleanup_library(self, library_name: str):
```

Comment:
Can just remove this now - this is how it's defined on the base `arctic_library_adapter`. In fact, I think we can drop the `cleanup_library` method completely now